cwltool/process.py

```diff
@@ -596,13 +596,15 @@ def __init__(self,
 
         dockerReq, is_req = self.get_requirement("DockerRequirement")
 
-        if dockerReq and dockerReq.get("dockerOutputDirectory") and not is_req:
+        if dockerReq is not None and dockerReq.get("dockerOutputDirectory")\
+                and is_req is not None and not is_req:
```

Comment:
while you're at it, might as well say `"dockerOutputDirectory" in dockerReq` instead of the `.get()`

Reply:
Not sure why github doesn't mark this as outdated, it's fixed.